src/veh_appliance.cpp

```diff
@@ -61,9 +64,10 @@ void place_appliance( const tripoint &p, const vpart_id &vpart, const std::optio
 
     veh->add_tag( flag_APPLIANCE );
 
+    int partnum = -1;
     if( base ) {
         item copied = *base;
-        veh->install_part( point_zero, vpart, std::move( copied ) );
+        partnum = veh->install_part( point_zero, vpart, std::move( copied ) );
```

Comment:
the else below should also set partnum for consistency

Reply:
followed the suggestion